android/src/main/java/com/brentvatne/react/ReactVideoPackage.java

```diff
@@ -25,9 +26,12 @@ public ReactVideoPackage(ReactExoplayerConfig config) {
 
     @Override
     public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
-        return Collections.singletonList(
-                new VideoDecoderPropertiesModule(reactContext)
-        );
+        List<NativeModule> modules = new ArrayList<NativeModule>();
```

Comment:
was their an issue here ?

Reply:
With `Collections.singletonList` i could not add `VideoManagerModule` to list. This implementation is from react-native [docs](https://reactnative.dev/docs/native-modules-android?android-language=java)